provider/testdata/data/json/datetime/gregory_skltns@1/und.json

```diff
@@ -0,0 +1,35 @@
+{
+  "y": "y",
+  "yM": "y-MM",
+  "yMd": "y-MM-dd",
+  "yMdE": "y-MM-dd, E",
+  "yMMM": "y MMM",
+  "yMMMd": "y MMM d",
+  "yMMMdE": "y MMM d, E",
+  "yMMMM": "y MMMM",
+  "M": "L",
+  "Md": "MM-dd",
+  "MdE": "MM-dd, E",
+  "MMM": "LLL",
+  "MMMd": "MMM d",
+  "MMMdE": "MMM d, E",
+  "MMMMd": "MMMM d",
+  "d": "d",
+  "dE": "d, E",
+  "E": "ccc",
+  "Ehm": "E h:mm a",
+  "Ehms": "E h:mm:ss a",
+  "EHm": "E HH:mm",
+  "EHms": "E HH:mm:ss",
+  "h": "h a",
+  "hm": "h:mm a",
+  "hms": "h:mm:ss a",
+  "hmsv": "h:mm:ss a v",
+  "hmv": "h:mm a v",
+  "H": "HH",
+  "Hm": "HH:mm",
+  "Hms": "HH:mm:ss",
+  "Hmsv": "HH:mm:ss v",
+  "Hmv": "HH:mm v",
+  "ms": "mm:ss"
+}
```

Comment:
Question: I'm commenting on `provider/testdata/data/testdata.postcard`, but it won't let me comment on a binary file. Why is there a 601 byte increase here? I guess it's not huge considering the file is already 571K, but I think it would be good to know why the increase, and if we should be concerned.

Reply:
@Manishearth is looking into it.